server/bin/pbench-server-erase.sh

```diff
@@ -0,0 +1,69 @@
+#! /bin/bash
+
+# Erase Pbench Server 1.0 state completely. This is a simple and "brute force"
+# sequence with few "blade guards", so be careful! (You have been warned!)
+#
+# 1. Delete the Elasticsearch indices and templates using the configured
+#    prefix value;
+# 2. Delete the configured PostgreSQL database;
+# 3. Delete the ARCHIVE, INCOMING, and RESULTS file system trees
+#
+# NOTE: The Pbench server should be *stopped* previously as root, and this
+# script should be run under `su --login pbench`.
```

Comment:
The real point was intended to be that, because of the way we have the Pbench Server installation set up, you should run *this* script under the `pbench` account, which means I can't realistically automate the server shutdown/restart as *that* needs to be under `root`. Clearly I missed the target here.